bundles/org.openhab.binding.siemensrds/src/main/java/org/openhab/binding/siemensrds/internal/RdsDataPoints.java

```diff
@@ -324,6 +324,15 @@ public boolean refresh(String apiKey, String token) {
                         set.add(String.format("\"%s\"", pointId));
                     }
                 }
+
+                for (Map.Entry<String, BasePoint> entry : points.entrySet()) {
+                    BasePoint point = entry.getValue();
+                    if (point != null && point.memberName != null && point.memberName.equals("Online")) {
```

Comment:
Position literals first in `String` comparisons - that way if the `String` is `null` you won't get a `NullPointerException`, it'll just return `false`. [source: PMD](http://pmd.sourceforge.net/pmd-4.3.0/rules/design.html#PositionLiteralsFirstInComparisons)

Reply:
Hey @martinvw many thanks for the coding suggestion; I will definitely use it in future.